examples/webgl_camera_array2.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js webgl - arraycamera</title>
```

Comment:
Instead of adding a new example, can you update `webgpu_camera_array` instead? I think it's fine if `webgpu*` might set `forceWebGL: true` to `true`. This will also be `true` for upcoming `webgpu_xr*` demos.

In this way, we don't have three permutations of the same demo, just two (one for each renderer).

Reply:
> It also seems to me that WebGPU will at some point support WebXR?

AFAIK, they are currently working on that: https://github.com/immersive-web/WebXR-WebGPU-Binding